website/content/docs/connect/transparent-proxy.mdx

```diff
@@ -162,25 +162,34 @@ configure exceptions on a per-Pod basis. The following Pod annotations allow you
-transparent proxy's datacenter. Services can also dial explicit upstreams in other datacenters without transparent proxy, for example, by adding an
-[annotation](/docs/k8s/annotations-and-labels#consul-hashicorp-com-connect-service-upstreams) such as
-`"consul.hashicorp.com/connect-service-upstreams": "my-service:1234:dc2"` to reach an upstream service called `my-service`
-in the datacenter `dc2`.
-* In the deployment configuration where a [single Consul datacenter spans multiple Kubernetes clusters](/docs/k8s/installation/deployment-configurations/single-dc-multi-k8s), services in one Kubernetes cluster must explicitly dial a service in another Kubernetes cluster using the [consul.hashicorp.com/connect-service-upstreams](/docs/k8s/annotations-and-labels#consul-hashicorp-com-connect-service-upstreams) annotation. An example would be
-`"consul.hashicorp.com/connect-service-upstreams": "my-service:1234"`, where `my-service` is the service that exists in another Kubernetes cluster and is exposed on port `1234`. Although Transparent Proxy is enabled, KubeDNS is not utilized when communicating between services existing on separate Kubernetes clusters.
-
-* When dialing headless services, the request will be proxied using a plain TCP
- proxy. The upstream's protocol is not considered.
+* In a deployment configuration with [federation between Kubernetes clusters](/docs/k8s/installation/multi-cluster/kubernetes),
+  services in one Kubernetes cluster must explicitly dial a service in another Kubernetes cluster (Consul datacenter) using the
+  [consul.hashicorp.com/connect-service-upstreams](/docs/k8s/annotations-and-labels#consul-hashicorp-com-connect-service-upstreams)
+  annotation. You cannot use transparent proxy. For example, an annotation of
+  `"consul.hashicorp.com/connect-service-upstreams": "my-service:1234:dc2"` will reach an upstream service called `my-service`
+  on port `1234` in the datacenter `dc2`.
+ 
+* In the deployment configuration with a
+  [single Consul datacenter spanning multiple Kubernetes clusters](/docs/k8s/installation/deployment-configurations/single-dc-multi-k8s),
+  services in one Kubernetes cluster must explicitly dial a service in another Kubernetes cluster using the
+  [consul.hashicorp.com/connect-service-upstreams](/docs/k8s/annotations-and-labels#consul-hashicorp-com-connect-service-upstreams)
+  annotation. You cannot use transparent proxy. For example, an annotation of
+  `"consul.hashicorp.com/connect-service-upstreams": "my-service:1234"`,
+  will reach an upstream service called `my-service` in another Kubernetes cluster and on port `1234`.
+  Although transparent proxy is enabled, Kubernetes DNS is not utilized when communicating between services existing on separate Kubernetes clusters.
+  
+* In a deployment configuration with [cluster peering](/docs/connect/cluster-peering), you can use transparent proxy.
```

Comment:
```suggestion
* In a deployment configuration with [cluster peering](/docs/connect/cluster-peering), you can use transparent proxy and thus dialing services explicitly is not required. 
```

Reply:
This doesn't look quite right per the docs at https://www.consul.io/docs/connect/cluster-peering/k8s to add `"consul.hashicorp.com/transparent-proxy": "false"` and `"consul.hashicorp.com/connect-service-upstreams"` for dialing an imported service?

Reply:
@joatmon08 Would you be be open to also make changes on the doc @mikemorris pointed to show that TProxy is supported?

Reply:
@boruszak I agree, it is pretty confusing. I put it into a new section and for repetition, noted that federation + multi-cluster needs to use tproxy in upstreams. Additional review appreciated.

@david-yu Updated cluster peering doc to include tproxy.